docs/guides/configure-error-mitigation.mdx

```diff
@@ -58,7 +58,9 @@ applied at each resilience level.
 
 ## Configure Estimator with resilience levels
 
-You can use resilience levels to specify error mitigation techniques, or you can set custom techniques individually as described in [Custom error settings.](#advanced-error) You cannot specify resilience levels in Sampler. However, you can set custom techniques individually.
+You can use resilience levels to specify error mitigation techniques, or you can set custom techniques individually as described in [Custom error settings.](#advanced-error) 
+
+Sampler does not support error mitigation, but you can use the [mthree](https://qiskit.github.io/qiskit-addon-mthree/) (matrix-free measurement mitigation) package to do error mitigation locally.
```

Comment:
```suggestion
Sampler does not support error mitigation, but you can use the [mthree](https://qiskit.github.io/qiskit-addon-mthree/) (matrix-free measurement mitigation) package to perform error mitigation locally.
```

Reply:
I'm not a giant fan of "perform" either, but "do" sounds weird to me.

Reply:
I definitely like `perform` better than `do`. 👍